docs/dashboard/layout.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: docs-dashboard
+toc: toc-dashboard.html
+title: Layouting Dashboard
+---
+
+<!-- This document contains original Japanese description in comments -->
+
+### Basic UI Elements of Dashboard
+
+<!-- ### Dashboard画面の基本部品 -->
+
+<!--　Node-RED Dashboardは`Widget`、`Group`、および、`Tab`から構成されます。WidgetはButton、ChartなどGUI部品です。TabはWebブラウザ上に表示するダッシュボード画面全体を表します。Tabは複数定義でき、サイドメニューからの選択、もしくは、ui_controlノードで切り替えて一つのタブを表示できます。Groupは複数のWidgetを配置する領域を定義するために使用します。GroupおよびTabはConfigノードです。-->
+
+  The Node-RED Dashboard consists of `Widget`,`Group`, and `Tab`.  Widget is a GUI element such as button or chart.  Tab represents the entire dashboard screen displayed on the Web browser.  Multiple Tabs can be defined, and one tab can be displayed by selecting from the side menu or switched with the `ui_control node`.  Group is used to define the area where multiple widgets will be placed.  Group and Tab are defined as configuration nodes.
```

Comment:
I would keep the logical order. So once you introduce widgets, groups and then tabs. I would then follow with what a widget is (as you have) , then groups...  and then tabs